apps/system/utils/secure_log_cmd.c

```diff
@@ -0,0 +1,83 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/****************************************************************************
+ * Preprocessor Definitions
+ ****************************************************************************/
+#define LOG_SS_SLOT_PATH        "ss/32"
+
+/********************************************************************************
+ * Private Function Prototypes
+ ********************************************************************************/
+
+static void secure_log(int argc, char **args)
+{
+        security_handle hnd;
```

Comment:
How about move setting ss storege code to os/kernel/debug/secure_log.c and just call setting api here?

ss read is called here and os/kernel/debug/secure_log.c.
so how about  call ss function in only one directory?